package.json

```diff
@@ -64,11 +64,11 @@
     "mocha": "^7.0.0"
   },
   "dependencies": {
```

Comment:
Just as a matter of consistency, could you move `dependencies` above `devDependencies`?